lib/gooddata/lcm/lcm2.rb

```diff
@@ -255,13 +254,12 @@ def print_action_result(action, messages)
           end
         end
 
-        puts "\n#{table}"
+        GoodData.logger.info("\n#{table}")
       end
 
       def print_actions_result(actions, results)
         actions.each_with_index do |action, index|
           print_action_result(action, results[index])
-          puts
```

Comment:
Are you aware of the removed newline? Are you sure it will not break the output?

Reply:
Newline added.